Source/WebCore/platform/graphics/DestinationColorSpace.cpp

```diff
@@ -140,6 +138,26 @@ bool DestinationColorSpace::supportsOutput() const
 #endif
 }
 
+bool DestinationColorSpace::isXDR() const
+{
+#if USE(CG)
+    return CGColorSpaceUsesExtendedRange(platformColorSpace());
+#else
+    notImplemented();
+    return false;
+#endif
+}
+
+bool DestinationColorSpace::isHDR() const
```

Comment:
Rather than using `HDR`, maybe we should just talk about `ITUR_2100TF` perhaps with a nicer name.

Reply:
I renamed it to `usesRec2100TransferFunctions()` to be compatible with CG function name.